platform/platform-sample-package.md

```diff
@@ -0,0 +1,67 @@
+---
+title: Platform Sample Installation
+summary: Installation instructions for using the platform sample for demo purposes.
+component: PlatformSample
+reviewed: 2019-06-16
+---
+
+The Platform Sample can be used to demonstrate the platform without actually fully installing it. This is the easiest way to get some experience to learn the Particular Service Platform. It is also possible to include and show the platform in presentations or samples. It is also possible to (temporarily) include it in a custom Visual Studio solution for demonstration purposes.
+
+DANGER: The Platform Sample is not for production usage and should only be used for demonstrating it during presentations or in samples.
+
+The Platform Sample package includes the binaries of ServiceControl and ServiceControl.Monitoring and a self-hosted version of ServicePulse. All are configured to use the LearningTransport and don't support any other transports.
+
+## Sample
```

Comment:
Shouldn't this section come after installation? Seems like that should be the focus of this article.